app/utils.py

```diff
@@ -4,16 +4,35 @@
 from enum import Enum
 from types import GeneratorType
 from boardgamegeek import BGGClient, CacheBackendSqlite
-from schema import Game
+from app.schema import Game
 import datetime
 import json
+from itertools import islice
+
+
+def take(n, iterable):
+    "Return first n items of the iterable as a list"
+    return list(islice(iterable, n))
 
 
 def isoformat(o):
+    "returns input datetime in isoformat"
     return o.isoformat()
 
 
+def update_schema(game):
+    "takes a game with old scheme (lowercase props) and renames (uppercase)" "renames year_published to Year"
+    return {
+        "Name": game['name'],
+        "Id": game['id'],
+        "Fetched": game['fetched'],
+        "Developers": game['developers'],
+        "Year": game['year_published']
```

Comment:
You might consider using pydantic aliases to get year_published -> Year behavior at the schema level, since everything else is a pretty rote conversion.